ui/src/containers/Header/index.tsx

```diff
@@ -30,26 +31,51 @@ const Header: React.FC = observer(() => {
   const { user } = useMst();
   const history = useHistory();
   const [isModalOpen, setIsModalOpen] = React.useState(false);
+  const [isInstructionModallOpen, setInstructionModalOpen] = React.useState(false);
 
   const headerTools = (
     <PageHeaderTools>
       <Grid>
-        <GridItem span={10}>
-          <Search />
+        <GridItem span={8}>
+          <Grid>
+            <GridItem offset={1} span={10}>
```

Comment:
Why are we using offset here ??

Reply:
Also do we need so many grid items ??

Reply:
Also do we need grid inside a grid ??

Reply:
offset is for spacing and grid inside grid would help in managing block spaces better way , not needed but it's a good appraoch